### DISCUSSION TITLE
About spot color opacity (composed of Lab)

### DISCUSSION THREAD

**Discussion by diuming** [colour-science/colour#1025] [General]
### Question

I have a question about CIE Lab color.
How do I calculate percentage of spot color(composed of CIE Lab)?

for example,
|  |  L | a | b | Status | Opacity |
| ------: | -----: | ----: | -----: | ----: | ----: |
| 100%  | 55.6 | 2.1  | -40.4 | | 0 |
| 75%  | 71.5 | 0.2  | -24.6 | computed | 25 |
| 40% | 83.1 | -0.7 | -14.8 | computed | 60 |
| 30% | 87.4 | -0.6 | -10.9 | computed | 70 |
| 20% | 91.7 | -0.5 | -7.2 | computed | 80 |
| 10% | 95.9 | -0.3 | -3.5 | computed | 90 |

How do I calculate 50% Lab values from 100% Lab values?
Anyone could give me advice?

thanks

**Comment by thomasmansencal** [comment]
Hi @diuming,

Sorry, I don't understand what the computation is for here.

Cheers,

Thomas
